src/main/java/by/onliner/util/Assertions.java

```diff
@@ -0,0 +1,11 @@
+package by.onliner.util;
+
+/***
+ * The class is used to give us more assertions
+ */
+public class Assertions {
+
+    public static boolean assertContain (String expected, String contains) {
```

Comment:
Nice idea!
have you checked existing solutions for extending assertions?
Please update the name to 'assertContains' and add an assertion message to the method.
Also, add a Java Doc to this method.

Reply:
Add a Java Doc to this method.
Renamed the method.